### PR TITLE
feat: Add BarterYardClub werewolf collection

### PR DESCRIPTION
**Pull request by AlexandreBarbier**
Add Barter Yard Club werewolves collection

### CODE REVIEW

File: src/cadence/mainnet/testGetNFTs.sh [hunk 1]
```diff
@@ -216,6 +216,11 @@ BarterYard)
   flow scripts execute getNFTs.cdc --args-json '[{ "type": "Address", "value": "0xa95b021cf8a30d80" }, { "type": "Dictionary", "value": [{ "key": { "type": "String", "value": "BarterYardPack" }, "value": { "type": "Array", "value": [{ "type": "UInt64", "value": "365" }] } }] }]' --network mainnet
   ;;
 
+BarterYardClubWerewolf)
```
Comment (niveda-krish) [inline src/cadence/mainnet/testGetNFTs.sh:219]: Seeing the error below on running the test:
❌ Invalid argument: failed to execute script: failed to execute script at block (53d71d0ceb6af480030ca88fcc222a0a5cc8ad4661bec81cec360abd9cd82325): [Error Code: 1101] cadence runtime error Execution failed:
error: pre-condition failed: NFT not in collection
   --> 28abb9f291cadaf2.BarterYardClubWerewolf:453:26

**Comment by niveda-krish** [comment]
Seeing the error below on running the test:
❌ Invalid argument: failed to execute script: failed to execute script at block (53d71d0ceb6af480030ca88fcc222a0a5cc8ad4661bec81cec360abd9cd82325): [Error Code: 1101] cadence runtime error Execution failed:
error: pre-condition failed: NFT not in collection
   --> 28abb9f291cadaf2.BarterYardClubWerewolf:453:26

File: src/cadence/mainnet/getNFTs.cdc [hunk 2]
```diff
@@ -2327,6 +2329,52 @@ pub fun getBarterYardPack(owner: PublicAccount, id: UInt64): NFTData? {
   )
 }
```
Comment (niveda-krish) [inline src/cadence/mainnet/getNFTs.cdc:2331]: Could you add a comment with the location to your contract, eg https://flow-view-source.com/testnet/account/0xfafb022e4e45634b/contract/SturdyItems

Comment (AlexandreBarbier) [reply src/cadence/mainnet/getNFTs.cdc:2331]: Done

File: src/cadence/mainnet/testGetNFTs.sh (hunk 1 above)
Comment (AlexandreBarbier) [reply src/cadence/mainnet/testGetNFTs.sh:219]: fixed ;-)

**Review by prpatel05 — APPROVED** [review]

**Pull request merged**